### ISSUE TITLE
[Dungeon] Dire Maul: East - Warpwood Pod Spawning

### ISSUE THREAD

**Issue by Santah62** [KurthosWoW/TrinityCore#89]
**Description:**

Right next to Pusillin, there is a warpwood pod, that spawns a metric shitload of Whip Lashers

**Current behaviour:**

https://www.youtube.com/watch?v=PP6vCOPloKQ&feature=youtu.be

**Expected behaviour:**

They should spawn Whip Lashers only once, or atleast much slower than shown

**Comment by HelloKitty** [comment]
Yes, none of the pods seem to work. @Cillez seems to have addressed it https://github.com/renegadewow/Renegade-WoW/issues/82

@natnat201193 found the TC issue related to it too. https://github.com/TrinityCore/TrinityCore/issues/19371

**Comment by HelloKitty** [comment]
@Cillez 
Shauren is probably correct I assume because WoW dev was likely influenced by the exploitation in EQ where you could read process memory or packets to determine what was going on. Limiting the available information, especially for something you might want to consider a trap, and keeping it server-side only makes sense.

I would like to implement the full behavior. Whatever that was, so just disabling them might work temporarily but we'll definitely need to revisit the issue at some point.
